app/src/main/java/com/example/mayankjindal/voicecalculator/Splash.java

```diff
@@ -31,7 +23,6 @@ public void run() {
                 i = new Intent(getBaseContext(), MainActivity.class);
                 startActivity(i);
             }
-        }, 2000);
+        }, 0);
```

Comment:
@rishabhdeepsingh Remove the complete code related to splash rather than setting the time to 0.